addons/wowl/static/src/crash_manager/error_dialogs.ts

```diff
@@ -12,43 +12,86 @@ function capitalize(s: string | undefined): string {
 }
 
 interface ErrorDialogProps {
-  error: {
-    message?: string;
-    data?: {
-      debug?: string;
-    };
-    subType?: string;
-    traceback?: string;
-    type: "server" | "script";
-  };
+  name?: string;
+  message?: string;
+  traceback?: string;
 }
 
 export class ErrorDialog extends Component<ErrorDialogProps, OdooEnv> {
   static template = "wowl.ErrorDialog";
   static components = { Dialog };
-  title: string;
-  traceback: string;
+  title = this.env._t("Odoo Error");
+
+  state = useState({
+    showTraceback: false,
+  });
+
+  onClickClipboard() {
+    odoo.browser.navigator.clipboard.writeText(
+      `${this.props.name}\n${this.props.message}\n${this.props.traceback}`
+    );
+  }
+}
+
+export class ClientErrorDialog extends ErrorDialog {
```

Comment:
no need for this class, nor the next 2 classes. You can simply use a prop for the dialog manager service

Reply:
True unless alternativeComponent is kept. Maybe you want to throw an Error used once, so you don't want it to be in the switch in the crash manager. But you want it to have the "Odoo Server Error" title:
```js
const error = new OdooError("SOME_UNIQUE_ERROR_BUT_IT_IS_SERVER_RELATED")
error.alternativeComponent = ServerErrorDialog;
```
If alternativeComponent, then we can use props instead of new classes. Though I like having small classes, I guess it's preferences.